apps/user/models/role.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type Role struct {
+	gorm.Model
+	Name string `gorm:"type:varchar(30)"`
```

Comment:
We should have a varchar of 64 length as the role name could be longer.

Reply:
done